frontend/src/components/Diff/Diff.tsx

```diff
@@ -52,23 +52,24 @@ function FormatDiffText({ texts, highlighter }: {
     highlighter: Highlighter
 }) {
     let index = 0
-    return <> {
+    return <>{
```

Comment:
this space actually appears in the output, and I think we want it there. Or some replacement CSS